File: sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/SignalRServiceExtension/Bindings/SignalRConnectionInfo.cs
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.

using Newtonsoft.Json;

namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
{
    public class SignalRConnectionInfo
    {
        [JsonProperty("url")]
        public string Url { get; set; }

        [JsonProperty("accessToken")]
        public string AccessToken { get; set; }
    }
}

File: sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/binding-library/java/src/main/java/com/microsoft/azure/functions/signalr/SignalRConnectionInfo.java
/**
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License. See License.txt in the project root for
 * license information.
 */
package com.microsoft.azure.functions.signalr;

/**
 * <p>
 * SignalR connection information (used with SignalRConnectionInfo input binding)
 * </p>
 *
 * @since 1.0.0
 */
public class SignalRConnectionInfo {
    /**
     * SignalR Sevice endpoint
     */
    public String url;

    /**
     * Access token to use to connect to SignalR Service endpoint
     */
    public String accessToken;
}